frontend/integration-tests/tests/metalkube/dashboard.scenario.ts

```diff
@@ -0,0 +1,65 @@
+import { browser } from 'protractor';
+const execSync = require('child_process').execSync;
+
+import { appHost } from '../../protractor.conf';
+import * as dashboardView from '../../views/metalkube/dashboards.view';
+
+async function compareCounter(elem, expectedValue) {
+  const displayedValue = Number(await dashboardView.getTextIfPresent(elem, '0'));
+  expect(displayedValue).toEqual(expectedValue);
+}
+
+describe('Inventory card', () => {
+  beforeAll(async() => {
+    await browser.get(`${appHost}/dashboards`);
+    await dashboardView.isLoaded();
+    // wait until the counters in the inventory card show up
+    while (!(dashboardView.inventoryNodesUpCounter.isPresent() || dashboardView.inventoryNodesDownCounter.isPresent()) ||
```

Comment:
The negations are quite hard to read. Are we waiting for all the counters to be present?
I think ExpectedConditions is the way to go
```
await browser.wait(
  until.and(
    dashboardView.inventoryNodesUpCounter.isPresent(),
    dashboardView.inventoryNodesDownCounter.isPresent(),
    dashboardView.inventoryHostsUpCounter.isPresent(),
    dashboardView.inventoryHostsDownCounter.isPresent(),
  ), 15000 // the timeout that we're willing to wait for all these elements
);